plugins/agent-utilities/skills/fusion-parametric-design/src/fusion_design/mesh_fitting.py

```
@@ -762,29 +785,80 @@ def mesh_winding_evidence(vertices: Any, triangles: Any) -> dict[str, Any]:
+    # material verdicts. So the shells are separated over the manifold edges
+    # and their signs compared: they agree, and the global sign is one solid's,
+    # or they do not, and this mesh carries no global winding at all.
+    parent = list(range(len(faces)))
+
+    def _root(index: int) -> int:
+        while parent[index] != index:
+            parent[index] = parent[parent[index]]
+            index = parent[index]
+        return index
+
+    for key, members in incident.items():
```

Ruff (0.16.1)

[warning] Loop control variable `key` not used within loop body

(B007)